### ISSUE TITLE
Omitting overlay param in QRView() prevents widget from working...

### ISSUE THREAD

**Issue by iamshaunwu** [juliuscanute/qr_code_scanner#245]
iPhone 12 Pro Max / iOS 14.3 / qr_code_scanner: ^0.3.1
- By omitting **overlay: QrScannerOverlayShape()** in QRView(). I was able to get the widget working with the fullscreen as a view.

```
QRView(
            key: qrKey,
            onQRViewCreated: _onQRViewCreated,
            formatsAllowed: const <BarcodeFormat>[BarcodeFormat.qrcode],
          ),
```

- After upgrading to iOS 14.4, the widget does not work anymore
- Tried downgrading to qr_code_scanner: ^0.3.0, the widget still doesn't work.

**Comment by juliansteenbakker** [comment]
So this issue only persists on iOS 14.4? I will try to get a hold of an iOS 14.4 device, but i don't have one available right now. I tested omitting this on Android and it works just fine.

**Comment by juliansteenbakker** [comment]
So i've got an iPad and tested it and it seems that function `updateDImensions` is checking `overlay.cutOutSize` and `overlay.cutOutBottomOffset` while overlay is null. Since this function is only called on iOS its only causing problems on iOS. I'll patch this asap. Thank you for reporting this.

**Comment by juliansteenbakker** [comment]
I have released version 0.3.2.

**Comment by iamshaunwu** [comment]
Thanks! Working good on v0.3.2!
